### PR TITLE
Request flatcar 2765+ in next releases

### PR DESCRIPTION
**Pull request by stone-z**
Towards https://github.com/giantswarm/giantswarm/issues/16533 to keep the updates rolling

### CODE REVIEW

File: aws/requests.yaml [hunk 1]
```diff
@@ -16,6 +16,11 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: "> 14.1.1"
```
Comment (MarcelMue) [inline aws/requests.yaml:19]: WDYT about including it in 15.0.0 instead?

Comment (stone-z) [reply aws/requests.yaml:19]: For AWS and Azure this is actually only a bump of 2 release versions, so not as big a jump as it looks. I'm also fine with 15, just trying to keep new stuff as fresh as possible and cut down the tail a little bit

Comment (MarcelMue) [reply aws/requests.yaml:19]: I am just concerned if we will get pinged if something goes wrong and teams are only trying to release patches as everyone is focussed on CAPI.

**Review by MarcelMue — APPROVED** [review]
I am fine either way - both ways have pros and cons

File: aws/requests.yaml (hunk 1 above)
Comment (stone-z) [reply aws/requests.yaml:19]: They can always add an exception if it blows up, no harm done

**Pull request merged**